third_party/gpus/cuda/platform.bzl

```diff
@@ -0,0 +1,39 @@
+CUDA_VERSION = ""
```

Comment:
(can you use two space indents in this file for consistency with the other .bzl files? sorry :(